src/axom/mint/docs/sphinx/sections/getting_started.rst

```diff
@@ -203,9 +272,9 @@ Within the body of the kernel, the centroid of the cell is calculated by
 averaging the node coordinates. The code loops over the columns of the
```

Comment:
The note a few lines up doesn't seem to apply any more:

> Since in this kernel the nodeIds are not used, the third kernel argument is annotated with the AXOM_NOT_USED macro to silence compiler warnings.

Reply:
Yes: for accuracy, please remove the note at lines 243--246.

Reply:
Good catch! Thanks.